app/windows/runner_panel.py

```diff
@@ -404,3 +404,9 @@ def finished(self) -> None:
 
         # Cleanup process
         self.process.terminate()
+        answer = show_dialogue_conditional(
+            text="Operation complete, you can close the window.",
+            button_text_override=["Close Window"],
+        )
+        if answer == "Close Window":
+            self.close()
```

Comment:
I think show_dialogue_conditional doesn't allow you to override the cancel button text. Try the following with BinaryChoiceDialog. Also, add a title and rename operation to process for consistency with the phrasing used in the runner panel logs.

```suggestion
        diag = BinaryChoiceDialog(
            title="Process Complete",
            text="Process complete, you can close the window.",
            positive_text="Close Window",
            negative_text="Ok",
        )
        if diag.exec_is_positive():
            self.close()
```

Reply:
ok i just used your suggestion and done with it

also seems show_dialogue_input is depriciated ?
![image](https://github.com/user-attachments/assets/7d3a7d76-c722-4349-8235-559c46b8ddeb)

Reply:
Yes. That is intended. It's just a wrapper around QInputDialog right now and doesn't add anything new.